.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+      - ci
+      - 'release/**'
```

Comment:
Probably we want to also do these runs on PRs as well, see https://github.com/obsproject/obs-studio/actions/runs/830902393/workflow#L3-L15 as an example.

Reply:
Thought it'd do that automatically if I don't specify but I guess not.